TRIAGING.md

```diff
@@ -20,13 +20,16 @@ If you have an issue you'd like to bring forth please consider getting a link to
 
 ### Is there an agenda for each week?
 
-Meetings are lightly structured as follows:
+Meetings 60 minutes and structured as follows:
+
+1. Wait for quorum: As we gather and waiting for at least three maintainers to join, feel free to turn on video and have informal and open-to-all conversation.  Once quorum is reached, a volunteer will share their screen and proceed with the agenda. 
```

Comment:
What if quorum is never reached? maybe we only have 2 maintainers join. Are we scrapping the meeting? or are there just specific sections that we want to have with enough quorum?

Reply:
I think whoever is in the meeting can make a call what to do or not.  I wouldn't expect the same structure to be honored if this was the case.   Not sure I'd want to capture that in this agenda lines, or do you have a recommendation?

Reply:
I'm not sure we should put anything about quorum at all if we could start w/o reaching one. If this ends up reading "wait for quorum, but if quorum is not reached don't wait", then it's not saying much. If the intent behind this point is more of a "let's give folks a few mins to join", I'd either say that or just remove the paragraph altogether.